build-configs.yaml

```diff
@@ -631,10 +639,16 @@ build_configs:
               - 'allmodconfig'
               - 'allnoconfig'
           x86_64:
+            base_defconfig: 'x86_64_defconfig'
             extra_configs:
               - 'allmodconfig'
               - 'allnoconfig'
-            fragments: []
+
+      clang-10:
+        build_environment: clang-10
+        architectures:
+          arm64: *arm64_defconfig
+          x86_64: *x86_64_defconfig
```

Comment:
can we get multi_v7_defconfig for aarch32?

Reply:
Done.